### ISSUE TITLE
Moving reviewer comments process issue

### ISSUE THREAD

**Issue by calexh-sar** [sartography/spiff-arena#822]
I attempted to move reviewer_comments.bpmn from the [Budget Owner approval](https://dev.app.spiff.status.im/editor/process-models/misc:approval-strategy-main:budget-owner-approval/files/reviewer_comments.bpmn) process model to a new p;rocess model, [Reviewer Comments](https://dev.app.spiff.status.im/editor/process-models/misc:approval-strategy-main:reviewer-comments/files/reviewer_comments.bpmn).  Initially I:

1. Downloaded and then deleted the file from the Budget Owner approval process model.
2. Created the Reviewer Comments process model and uploaded the file.
3. Attempted to reestablish the association of the Call Activities calling this process 

The Call Activities continue to look for the initial location, both when I attempted to use the Search feature or run PP1.  Additionally, when I uploaded the file back into the Budget Owner approval process model, it allowed me to do so even though the process with the same name was still present in the Reviewer Comments process model.

**Comment by madhurrya** [comment]
I tested this scebario using the models given here.
https://dev.app.spiff.status.im/process-groups/misc:qa:call-activity-tests

Initially I had this 'secondary-model.bpmn' file inside the 'Call Activity Model 1' and it was called by all these 4 models and that file showed all these 4 as references.
![image](https://github.com/sartography/spiff-arena/assets/110527567/3e9c528f-0b30-4ac2-9348-98b05555243f)

Then I downloaded and deleted the 'secondary-model.bpmn' file from 'Call Activity Model 1' and added it inside this new 'Secondary Model Test'
![image](https://github.com/sartography/spiff-arena/assets/110527567/cf0653d7-2ed4-45a0-9255-bef9e4b397d1)

After that when I run the above 4 models, they run without any error.
But when I go to the https://dev.app.spiff.status.im/editor/process-models/misc:qa:call-activity-tests:secondary-model-test/files/secondary-model.bpmn file I don't see any references there.

![image](https://github.com/sartography/spiff-arena/assets/110527567/682c8ad4-e562-4e29-a172-c954fc40e343)

**Comment by madhurrya** [comment]
@calexh-sar do we need to create a new ticket for the above issue or should it be handled under this?

**Comment by calexh-sar** [comment]
I think fixing references as part of this issue would be appropriate unless @jbirddog would like it handled otherwise.

**Comment by jbirddog** [comment]
> I tested this scebario using the models given here. https://dev.app.spiff.status.im/process-groups/misc:qa:call-activity-tests
> 
> Initially I had this 'secondary-model.bpmn' file inside the 'Call Activity Model 1' and it was called by all these 4 models and that file showed all these 4 as references.

> 
> Then I downloaded and deleted the 'secondary-model.bpmn' file from 'Call Activity Model 1' and added it inside this new 'Secondary Model Test' 

> 
> After that when I run the above 4 models, they run without any error. But when I go to the https://dev.app.spiff.status.im/editor/process-models/misc:qa:call-activity-tests:secondary-model-test/files/secondary-model.bpmn file I don't see any references there.
> 

This has been fixed on main.

**Comment by jbirddog** [comment]
> The Call Activities continue to look for the initial location, both when I attempted to use the Search feature or run PP1. Additionally, when I uploaded the file back into the Budget Owner approval process model, it allowed me to do so even though the process with the same name was still present in the Reviewer Comments process model.

This has been fixed on main as well.

**Comment by madhurrya** [comment]
Followed the same steps I followed above earlier, using the new https://dev.app.spiff.status.im/editor/process-models/misc:qa:call-activity-tests:secondary-model-test/files/secondary-model-test2.bpmn file.
This time it worked as expected.

And when I tried to upload that same file to 'Call Activity Model 1' again it showed this validation message so I can't upload the same file again.
![image](https://github.com/sartography/spiff-arena/assets/110527567/74b6cd8b-3ca7-4785-b1db-345f638d291a)
